package.json

```diff
@@ -51,18 +51,22 @@
 		"framer-motion": "4.1.17",
 		"graphql-request": "3.4.0",
 		"i18next": "19.7.0",
+		"image-trace-loader": "^1.0.2",
+		"imagemin-mozjpeg": "^10.0.0",
 		"immer": "9.0.6",
+		"intercept-stdout": "^0.1.2",
+		"jimp": "^0.16.1",
```

Comment:
No worry about this, if there is no use for these packages, it will not affect the bundle size and app.

these packages are optional: https://github.com/cyrilwanner/next-optimized-images#optimization-packages

Reply:
going to test some more things out today on this note.

Reply:
Why do we need to remove `next-optimize-image`, it did optimization for `svg` files.

Reply:
> I dont see these image libraries being used as a loader or called anywhere, are these providing some functionality with this upgrade? can you expand on this?
> 
> referring to these:
> 
> ```
> "image-trace-loader": "^1.0.2"
> "imagemin-mozjpeg": "^10.0.0"
> "jimp": "^0.16.1"
> "svg-sprite-loader": "^6.0.11",
> "responsive-loader": "^2.3.0",
> ```

https://github.com/zhyd1997/kwenta/pull/2